marketing_automation/segment.rst

```diff
@@ -0,0 +1,81 @@
+=================================
+Segment the Database with Filters
+=================================
+Filters let you reach a granular level of detail in your records.
+
+How to work with filters
+========================
+When working with filters, you will see that some of the rules have an arrow button next to it,
+informing you that the parameter has more variables within it.
+
+
+.. image:: media/filters1.png
+   :align: center
+   :alt: Filters in Odoo Marketing Automation
+
+
+More than one parameter line - *node* (+) - can be added under what is called a *branch* (...).
+To exclude a *node* or a *branch*, click on *Delete node* (x).
+
+
+.. image:: media/filters2.png
+   :align: center
+   :alt: Filters in Odoo Marketing Automation
+
+
+Every time a new branch is created, you can choose if you would like the records to match *all*
+criteria of the upcoming rules, or *any*, meaning one rule or the other.
+
+
+Use cases
+=========
+**Scenario**: narrow the target down to new opportunities in the pipeline.
```

Comment:
Content nitpicking: I'd put each scenario in a separate bullet point with the image below aligned left to have a clearer split of the various use cases.